package.json

```diff
@@ -34,38 +31,38 @@
         "nav-frontend-lenker-style": "^0.2.24",
         "nav-frontend-paneler": "^2.0.4",
         "nav-frontend-paneler-style": "^0.3.21",
-        "nav-frontend-popover": "^0.0.28",
-        "nav-frontend-popover-style": "^0.0.5",
+        "nav-frontend-popover": "^0.0.29",
+        "nav-frontend-popover-style": "^0.0.6",
         "nav-frontend-skjema": "^3.0.17",
         "nav-frontend-skjema-style": "^2.0.8",
         "nav-frontend-spinner": "^2.0.2",
         "nav-frontend-spinner-style": "^0.2.5",
         "nav-frontend-tabell-style": "^0.2.19",
-        "nav-frontend-toggle": "^0.0.40",
-        "nav-frontend-toggle-style": "^0.0.35",
+        "nav-frontend-toggle": "^0.0.44",
+        "nav-frontend-toggle-style": "^0.0.36",
         "nav-frontend-typografi": "^2.0.26",
         "nav-frontend-typografi-style": "^1.0.22",
         "prop-types": "^15.7.2",
         "react": "^16.13.1",
         "react-collapse": "^5.0.1",
         "react-dom": "^16.12.0",
         "react-loading-skeleton": "^2.0.1",
-        "react-motion": "^0.5.2",
         "react-responsive": "^8.0.3",
         "react-router-dom": "^5.1.2",
-        "react-scripts": "^3.3.0",
         "react-to-print": "^2.10.0",
         "recharts": "2.0.0-beta.1"
     },
     "devDependencies": {
+        "@craco/craco": "^5.7.0",
+        "@types/amplitude-js": "^6.0.0",
```

Comment:
Lurer på hvorfor disse to avhengigheter flyttes til devDependencies?

Reply:
Alle @types-avhengigheter kan være i dev, siden det bare har noe å si når man utvikler. Jeg flyttet craco til dev fordi det var sånn i kontakt-oss, og det så ut til å fungere. Men i følge denne tråden kan det være lurt å ha det i dependencies likevel: https://github.com/gsoft-inc/craco/pull/23